edgelet/workload/src/models/status.rs

```diff
@@ -0,0 +1,85 @@
+/* 
+ * IoT Edge Module Workload API
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * OpenAPI spec version: 2020-07-22
+ * 
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+
+
+#[allow(unused_imports)]
+use serde_json::Value;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Status {
```

Comment:
is this struct used?

Reply:
It is contained in [`ModuleDetails`](https://github.com/R25l84IHeXjIxy6HO1QXn0y0Dq9mt8EN/iotedge/blob/4e56839623c254c02c914d5b33a5df6a2b55b4ec/edgelet/workload/src/models/module_details.rs#L29).